src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,35 @@
+package duke.command;
+
+import duke.gui.Ui;
+import duke.storage.Storage;
+import duke.task.TaskList;
+
+/**
+ * Manages the termination of a chat.
+ */
+public class ByeCommand extends Command {
+
+    private String userInput;
+
+    /**
+     * Public constructor that stores the input from the user.
+     *
+     * @param userInput the String input by the user
+     */
+    public ByeCommand(String userInput) {
+        this.userInput = userInput;
+    }
+
+    /**
+     * Executes the command and returns a closing message.
+     *
+     * @param ui the ui class that prints text in a readable format
+     * @param storage the storage object which handles reading and writing of data
+     * @param taskList the list of tasks currently stored
+     * @return a String which replies to the user
+     */
+    @Override
+    public String runCommand(Ui ui, Storage storage, TaskList taskList) {
```

Comment:
sorry a bit confused as to why the three params in this runCommand function are not used, are they not needed?